module/importer-ergonode-1/src/Infrastructure/Processor/ErgonodeImportProcess.php

```diff
@@ -59,6 +62,12 @@ public function start(Import $import): void
             foreach ($this->steps as $step) {
                 $step($import, $source, $zipDirectory);
             }
+        } catch (ErgonodeZipExtractorException $exception) {
+            $this->logger->warning($exception);
+            throw new ImportException('Invalid zip source');
+        } catch (ReaderFileProcessException $exception) {
+            $this->logger->warning($exception);
+            throw new ImportException(sprintf('Can\'t process file %s', $exception->getFilename()));
         } catch (Throwable $exception) {
```

Comment:
One more thing - as Rafał mentioned perhaps this should rethrow exception so the higher layer would take care of it